app/src/main/java/org/simple/clinic/home/report/ReportsModel.kt

```diff
@@ -4,4 +4,9 @@ import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
 
 @Parcelize
-class ReportsModel : Parcelable
+class ReportsModel : Parcelable {
+
+  companion object {
```

Comment:
Can you tell me why do we need this here?

Reply:
We are using it when creating `MobiusTestFixture` in `ReportsScreenControllerTest `